include/errors.h

```diff
@@ -188,601 +125,22 @@ enum generic_identifiers {
-#define SWO_APD_STA_1C                    (ERR_APD_STA + ERR_GEN_ID_1C)    // 0x661C
-#define SWO_APD_STA_1D                    (ERR_APD_STA + ERR_GEN_ID_1D)    // 0x661D
-#define SWO_APD_STA_1E                    (ERR_APD_STA + ERR_GEN_ID_1E)    // 0x661E
-#define SWO_APD_STA_1F                    (ERR_APD_STA + ERR_GEN_ID_1F)    // 0x661F
-#define SWO_APD_STA_20                    (ERR_APD_STA + ERR_GEN_ID_20)    // 0x6620
-#define SWO_APD_STA_21                    (ERR_APD_STA + ERR_GEN_ID_21)    // 0x6621
-#define SWO_APD_STA_22                    (ERR_APD_STA + ERR_GEN_ID_22)    // 0x6622
-#define SWO_APD_STA_23                    (ERR_APD_STA + ERR_GEN_ID_23)    // 0x6623
-#define SWO_APD_STA_24                    (ERR_APD_STA + ERR_GEN_ID_24)    // 0x6624
-#define SWO_APD_STA_25                    (ERR_APD_STA + ERR_GEN_ID_25)    // 0x6625
-#define SWO_APD_STA_26                    (ERR_APD_STA + ERR_GEN_ID_26)    // 0x6626
-#define SWO_APD_STA_27                    (ERR_APD_STA + ERR_GEN_ID_27)    // 0x6627
-#define SWO_APD_STA_28                    (ERR_APD_STA + ERR_GEN_ID_28)    // 0x6628
-#define SWO_APD_STA_29                    (ERR_APD_STA + ERR_GEN_ID_29)    // 0x6629
-#define SWO_APD_STA_2A                    (ERR_APD_STA + ERR_GEN_ID_2A)    // 0x662A
-#define SWO_APD_STA_2B                    (ERR_APD_STA + ERR_GEN_ID_2B)    // 0x662B
-#define SWO_APD_STA_2C                    (ERR_APD_STA + ERR_GEN_ID_2C)    // 0x662C
-#define SWO_APD_STA_2D                    (ERR_APD_STA + ERR_GEN_ID_2D)    // 0x662D
-#define SWO_APD_STA_2E                    (ERR_APD_STA + ERR_GEN_ID_2E)    // 0x662E
-#define SWO_APD_STA_2F                    (ERR_APD_STA + ERR_GEN_ID_2F)    // 0x662F
-#define SWO_APD_STA_30                    (ERR_APD_STA + ERR_GEN_ID_30)    // 0x6630
-#define SWO_APD_STA_31                    (ERR_APD_STA + ERR_GEN_ID_31)    // 0x6631
-#define SWO_APD_STA_32                    (ERR_APD_STA + ERR_GEN_ID_32)    // 0x6632
-#define SWO_APD_STA_33                    (ERR_APD_STA + ERR_GEN_ID_33)    // 0x6633
-#define SWO_APD_STA_34                    (ERR_APD_STA + ERR_GEN_ID_34)    // 0x6634
-#define SWO_APD_STA_35                    (ERR_APD_STA + ERR_GEN_ID_35)    // 0x6635
-#define SWO_APD_STA_36                    (ERR_APD_STA + ERR_GEN_ID_36)    // 0x6636
-#define SWO_APD_STA_37                    (ERR_APD_STA + ERR_GEN_ID_37)    // 0x6637
-#define SWO_APD_STA_38                    (ERR_APD_STA + ERR_GEN_ID_38)    // 0x6638
-#define SWO_APD_STA_39                    (ERR_APD_STA + ERR_GEN_ID_39)    // 0x6639
-#define SWO_APD_STA_3A                    (ERR_APD_STA + ERR_GEN_ID_3A)    // 0x663A
-#define SWO_APD_STA_3B                    (ERR_APD_STA + ERR_GEN_ID_3B)    // 0x663B
-#define SWO_APD_STA_3C                    (ERR_APD_STA + ERR_GEN_ID_3C)    // 0x663C
-#define SWO_APD_STA_3D                    (ERR_APD_STA + ERR_GEN_ID_3D)    // 0x663D
+#ifndef SWO_SEC_PIN_15
```

Comment:
Do you need the `#ifndef`? As the value is the same, there is probably no issue with redefining the same value?